helix-view/src/editor.rs

```diff
@@ -161,7 +174,7 @@ impl Editor {
         let id = if let Some(id) = id {
             id
         } else {
-            let mut doc = Document::load(path)?;
+            let mut doc = Document::load(path, Some(self.config_loader.clone()))?;
```

Comment:
There's no need to clone an arc around if you're just passing it as a reference (cloning then dropping results in extra operations to modify the refcount inside the Arc). `&self.config_loader` and then using `&Loader` on the type would work.